3.0-RELEASE-NOTES.md

```diff
@@ -102,3 +102,37 @@ bulk `updateOrCreate`, you could use `async.each` or `Promise.all` to
 repeatedly call `updateOfCreate` for each instance.
 
 See [related code change](https://github.com/strongloop/loopback-datasource-juggler/pull/889) here.
+
+## Removing strict: validate and strict: throw
+
+- `strict: 'validate'` and `strict: 'throw'` are removed in favor of `strict: true`.
+
+- `strict: true`'s behavior change:
```

Comment:
Please add an item explaining how `strict:false` is behaving now (i.e. no change).